.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: PR Preview
```

Comment:
Hm, could we name this workflow (+ the file that it lives in) something more descriptive? "CI" doesn't really tell us anything about what this workflow is actually doing

Reply:
Nevermind, I brought back a separate workflow as others suggested :)